Topics/User_Experience/User_Research.md

```diff
@@ -0,0 +1,102 @@
+
+Example: I want to study space finder application, I would create a user persona first for the UX
+
+Step 1: Define a user: our user is likely to be a student or working professional who is looking for a quiet, comfortable, and productive space to study or work.
+
+Step 2: Gather information: To create a user persona, we need to gather information about our target users' needs, goals, and behaviors. This could involve conducting surveys, interviews, or user testing. Some questions to consider include:
+
+- What type of user is most likely to use your app?
+
+- What features are most important to them in a study space? (e.g. wifi, power outlets, comfortable seating, quiet environment, etc.)
+
+- What are their pain points when it comes to finding a good study space? (e.g. lack of availability, noise, uncomfortable seating, etc.)
+
+Step 3: Create a profile: Based on the information you have gathered, create a profile of your user. This should include demographic information, such as age, gender, and occupation, as well as behavioral information, such as goals, needs, and pain points. For example:
+
+Name: Emily
+
+Age: 20
+
+Occupation: UTSG student
+
+Goals: Find a quiet and productive study space that is close to campus
+
+Needs: Comfortable seating, wifi, power outlets, quiet environment
+
+Pain Points: Limited availability of study spaces, noisy environments, uncomfortable seating
+
+
+2. User Testing: User testing involves observing and gathering feedback from users as they interact with your product. This can be done through various methods, such as in-person interviews, remote testing, or A/B testing. The goal of user testing is to identify usability issues, pain points, and opportunities for improvement.
+
+Example: We have finished our prototype for study space finder application, we need to conduct a user testing then.
+
+Step 1: Identify the testing goals: Before conducting user testing, you need to identify the goals I want to achieve. For example, I might want to test the ease of use of the app, the effectiveness of the search function, and the relevance of the search results.
+
+Step 2: Recruit participants: Next, I need to recruit participants for the user testing. To do this, I can reach out to your target user base, such as college students or working professionals who are looking for study spaces. I could also offer incentives, such as a gift card or free coffee, to encourage participation.
+
+Step 3: Create a testing plan: Once I have recruited participants, create a plan for the user testing. This should include a list of tasks for participants to complete, such as searching for a study space near a specific location, and a set of questions to gather feedback on the app's usability and functionality.
```

Comment:
I suggest switching the steps for recruiting participants and creating a testing plan. So you can immediately conduct the testing when you have the participants, recruiting participants when you don't even have a testing plan seems odd.